### PR TITLE
Added accessibility documentation for components

### LINKED ISSUE #1550: Add accessibility section for `select`
### Subject

accessibility section for `select`

#### Create an accessibility section referring to the links below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes ( https://github.com/yamada-ui/yamada-ui/issues/1552 )
https://www.radix-ui.com/primitives/docs/components/select#accessibility
https://mantine.dev/core/select/#accessibility
https://mui.com/material-ui/react-select/#accessibility

#### Include the following content
* keboard interaction
* aria, tabindex property are suported

#### reference
https://github.com/yamada-ui/yamada-ui/issues/1959

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Closes #1550

## Description

Added accessibility documentation for components.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by mei28 — DISMISSED** [review]

File: docs/contents/components/forms/multi-select/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+|                                        | `aria-required`           | Set to `"true"` when `isRequired` is set.                                            |
+| `SelectClearIcon` <Tag>Internal</Tag>  | `role="button"`           | Indicates that this is a button.                                                     |
+|                                        | `aria-label`              | `"Clear value"` is set.                                                              |
+| `MultiSelectList` <Tag>Internal</Tag>  | `role="listbox"`          | Indicates that this is a listbox.                                                    |
+|                                        | `id`                      | The `id` used to associate with `MultiSelectField`.                                  |
+|                                        | `aria-multiselectable`    | Indicates that multiple selection is possible.                                       |
+|                                        | `aria-hidden`             | Set to "true" when the listbox is open, "false" when closed.                         |
```
Comment (chida09) [inline docs/contents/components/forms/multi-select/accessibility.mdx:67]:
```suggestion
|                                        | `aria-hidden`             | Set to `"true"` when the listbox is open, `"false"` when closed.                         |
```

File: docs/contents/components/forms/select/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+|                                    | `aria-readonly`           | `isReadOnly`が設定されている場合は`"true"`を設定します。                                                      |
+|                                    | `aria-disabled`           | `isDisabled`が設定されている場合は`"true"`を設定します。                                                      |
+|                                    | `aria-invalid`            | `isInvalid`が設定されている場合は`"true"`を設定します。                                                       |
+|                                    | `aria-required`           | `isRequired`が設定されている場合は`"true"`を設定します。                                                      |
+| `SelectList` <Tag>内部</Tag>       | `role="listbox"`          | リストボックスであることを示します。                                                                          |
+|                                    | `id`                      | `SelectField`と関連付けるために使用する`id`。                                                                 |
+|                                    | `aria-hidden`             | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                         |
```
Comment (chida09) [inline docs/contents/components/forms/select/accessibility.ja.mdx:62]:
```suggestion
|                                    | `aria-hidden`             | リストボックスが開いている場合は`"true"`を設定し、閉じている場合は`"false"`を設定します。                         |
```

File: docs/contents/components/forms/select/accessibility.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+|                                        | `aria-readonly`           | Set to `"true"` when `isReadOnly` is set.                                       |
+|                                        | `aria-disabled`           | Set to `"true"` when `isDisabled` is set.                                       |
+|                                        | `aria-invalid`            | Set to `"true"` when `isInvalid` is set.                                        |
+|                                        | `aria-required`           | Set to `"true"` when `isRequired` is set.                                       |
+| `SelectList` <Tag>Internal</Tag>       | `role="listbox"`          | Indicates that this is a listbox.                                               |
+|                                        | `id`                      | The `id` used to associate with `SelectField`.                                  |
+|                                        | `aria-hidden`             | Set to "true" when the listbox is open, "false" when closed.                    |
```
Comment (chida09) [inline docs/contents/components/forms/select/accessibility.mdx:64]:
```suggestion
|                                        | `aria-hidden`             | Set to `"true"` when the listbox is open, `"false"` when closed.                    |
```

File: docs/contents/components/forms/multi-select/accessibility.ja.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+| `Escape`         | リストボックスを閉じます。                                                                           | `closeOnEsc={true}` |
+
+## ARIAロールと属性
+
+| コンポーネント                     | ロールと属性              | 使い方                                                                                                             |
+| ---------------------------------- | ------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| `MultiSelectField` <Tag>内部</Tag> | `role="combobox"`         | コンボボックスであることを示します。                                                                               |
+|                                    | `aria-haspopup="listbox"` | リストボックスが存在することを示します。                                                                           |
+|                                    | `aria-activedescendant`   | フォーカスされた項目の`id`を設定します。                                                                           |
+|                                    | `aria-expanded`           | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                              |
```
Comment (chida09) [inline docs/contents/components/forms/multi-select/accessibility.ja.mdx:54]:
```suggestion
|                                    | `aria-expanded`           | リストボックスが開いている場合は`"true"`を設定し、閉じている場合は`"false"`を設定します。                              |
```

File: docs/contents/components/forms/multi-select/accessibility.mdx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+| `Escape`         | Closes the listbox.                                                                               | `closeOnEsc={true}` |
+
+## ARIA Roles and Attributes
+
+| Component                              | Roles and Attributes      | Usage                                                                                |
+| -------------------------------------- | ------------------------- | ------------------------------------------------------------------------------------ |
+| `MultiSelectField` <Tag>Internal</Tag> | `role="combobox"`         | Indicates that this is a combobox.                                                   |
+|                                        | `aria-haspopup="listbox"` | Indicates that a listbox exists.                                                     |
+|                                        | `aria-activedescendant`   | Set to the `id` of the focused item.                                                 |
+|                                        | `aria-expanded`           | Set to "true" when the listbox is open, "false" when closed.                         |
```
Comment (chida09) [inline docs/contents/components/forms/multi-select/accessibility.mdx:56]:
```suggestion
|                                        | `aria-expanded`           | Set to `"true"` when the listbox is open, `"false"` when closed.                         |
```

File: docs/contents/components/forms/select/accessibility.ja.mdx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+| `Escape`         | リストボックスを閉じます。                                                                           | `closeOnEsc={true}`    |
+
+## ARIAロールと属性
+
+| コンポーネント                     | ロールと属性              | 使い方                                                                                                        |
+| ---------------------------------- | ------------------------- | ------------------------------------------------------------------------------------------------------------- |
+| `SelectField` <Tag>内部</Tag>      | `role="combobox"`         | コンボボックスであることを示します。                                                                          |
+|                                    | `aria-haspopup="listbox"` | リストボックスが存在することを示します。                                                                      |
+|                                    | `aria-activedescendant`   | フォーカスされた項目の`id`を設定します。                                                                      |
+|                                    | `aria-expanded`           | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                         |
```
Comment (chida09) [inline docs/contents/components/forms/select/accessibility.ja.mdx:54]:
```suggestion
|                                    | `aria-expanded`           | リストボックスが開いている場合は`"true"`を設定し、閉じている場合は`"false"`を設定します。                         |
```

File: docs/contents/components/forms/multi-select/accessibility.mdx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+description: >-
+  `MultiSelect` is a component used for allowing users to select multiple values
+  from a list of options.
+package_name: "@yamada-ui/select"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+The `MultiSelect` follows the[WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```
Comment (chida09) [inline docs/contents/components/forms/multi-select/accessibility.mdx:12]:
```suggestion
The `MultiSelect` follows the [WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```

File: docs/contents/components/forms/select/accessibility.mdx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+description: >-
+  `Select` is a component used for allowing a user to choose one option from a
+  list.
+package_name: "@yamada-ui/select"
+order: 3
+with_description: true
+tab: Accessibility
+---
+
+The `Select` follows the[WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```
Comment (chida09) [inline docs/contents/components/forms/select/accessibility.mdx:12]:
```suggestion
The `Select` follows the [WAI-ARIA - Combobox Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/) for accessibility.
```

File: docs/contents/components/forms/select/accessibility.ja.mdx (hunk 2 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/select/accessibility.ja.mdx:62]: Fixed.

File: docs/contents/components/forms/select/accessibility.ja.mdx (hunk 6 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/select/accessibility.ja.mdx:54]: Fixed.

File: docs/contents/components/forms/select/accessibility.mdx (hunk 8 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/select/accessibility.mdx:12]: Fixed.

File: docs/contents/components/forms/select/accessibility.mdx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+| `Escape`         | Closes the listbox.                                                                               | `closeOnEsc={true}`    |
+
+## ARIA Roles and Attributes
+
+| Component                              | Roles and Attributes      | Usage                                                                           |
+| -------------------------------------- | ------------------------- | ------------------------------------------------------------------------------- |
+| `SelectField` <Tag>Internal</Tag>      | `role="combobox"`         | Indicates that this is a combobox.                                              |
+|                                        | `aria-haspopup="listbox"` | Indicates that a listbox exists.                                                |
+|                                        | `aria-activedescendant`   | Set to the `id` of the focused item.                                            |
+|                                        | `aria-expanded`           | Set to "true" when the listbox is open, "false" when closed.                    |
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/select/accessibility.mdx:56]: Fixed.

File: docs/contents/components/forms/select/accessibility.mdx (hunk 3 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/select/accessibility.mdx:64]: Fixed.

File: docs/contents/components/forms/multi-select/accessibility.ja.mdx (hunk 4 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-select/accessibility.ja.mdx:54]: Fixed.

File: docs/contents/components/forms/multi-select/accessibility.ja.mdx [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+|                                    | `aria-required`           | `isRequired`が設定されている場合は`"true"`を設定します。                                                           |
+| `SelectClearIcon` <Tag>内部</Tag>  | `role="button"`           | ボタンであることを示します。                                                                                       |
+|                                    | `aria-label`              | `"Clear value"`を設定します。                                                                                      |
+| `MultiSelectList` <Tag>内部</Tag>  | `role="listbox"`          | リストボックスであることを示します。                                                                               |
+|                                    | `id`                      | `MultiSelectField`と関連付けるために使用する`id`。                                                                 |
+|                                    | `aria-multiselectable`    | 複数選択可能であることを示します。                                                                                 |
+|                                    | `aria-hidden`             | リストボックスが開いている場合は"true"を設定し、閉じている場合は"false"を設定します。                              |
```
[the remark this answers is not in the document]
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-select/accessibility.ja.mdx:65]: Fixed.

File: docs/contents/components/forms/multi-select/accessibility.mdx (hunk 7 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-select/accessibility.mdx:12]: Fixed.

File: docs/contents/components/forms/multi-select/accessibility.mdx (hunk 1 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/multi-select/accessibility.mdx:67]: Fixed.

**Review by chida09 — APPROVED** [review]

**Pull request merged**